diamond-build/config/definitions/diamond

```diff
@@ -0,0 +1,48 @@
+- job:
+    name: diamond-build
+    node: small && trusty
```

Comment:
would you mind doing the following to be a bit more flexible:

    small && (trusty || centos)